bin/console

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env php
```

Comment:
Did you remove this on purpose?
This should stay, it's not a true comment, it's interpreted by the CLI and meant to tell the path of the PHP exec file